Cython/Compiler/ExprNodes.py

```diff
@@ -7341,7 +7341,7 @@ def analyse_as_cimported_attribute_node(self, env, target):
         module_scope = self.obj.analyse_as_module(env)
         if module_scope:
             entry = module_scope.lookup_here(self.attribute)
-            if entry and (
+            if entry and not entry.known_standard_library_import and (
```

Comment:
I kind of where if this while if-statement can go away, but that's probably something to be investigated later